packages/lesswrong/components/bookmarks/FriendlyBookmarksFeed.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { registerComponent } from "@/lib/vulcan-lib/components";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { useCurrentUser } from "../common/withUser";
+import { UltraFeedObserverProvider } from "../ultraFeed/UltraFeedObserver";
+import { OverflowNavObserverProvider } from "../ultraFeed/OverflowNavObserverContext";
+import { useMulti } from "../../lib/crud/withMulti";
+import SingleColumnSection from "../common/SingleColumnSection";
+import SectionTitle from "../common/SectionTitle";
+import Loading from "../vulcan-core/Loading";
+import QuickTakesListItem from "../quickTakes/QuickTakesListItem";
+
+const styles = (theme: ThemeType) => ({
+  loading: {
+    marginTop: 32,
+  },
+  text: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 14,
+    fontWeight: 500,
+  },
+});
+
+const BookmarksFeed = ({classes}: {classes: ClassesType<typeof styles>}) => {
+  const currentUser = useCurrentUser();
+
+  const { results: bookmarks = [], loading, error } = useMulti({
+    collectionName: "Bookmarks",
+    fragmentName: "BookmarksFriendlyFeedItemFragment",
+    terms: {
+      view: "myBookmarkedComments",
+      userId: currentUser?._id,
+      limit: 20,
+    },
+    fetchPolicy: "network-only",
+    skip: !currentUser,
+  });
+
+  if (!currentUser || (loading && !bookmarks.length)) {
+    return (
+      <SingleColumnSection>
+        <Loading className={classes.loading} />
+      </SingleColumnSection>
+    );
+  }
+
+  if (error) {
+    return (
+      <SingleColumnSection>
+        <div className={classes.text}>Error loading saved comments</div>
+      </SingleColumnSection>
+    );
+  }
+
+  return (
+    <AnalyticsContext pageSectionContext="bookmarksFeed">
+      <UltraFeedObserverProvider incognitoMode={true} >
+      <OverflowNavObserverProvider>
+      <SingleColumnSection>
+        <SectionTitle title="Saved comments & quick takes" />
+        {bookmarks.map((bookmark) =>
+          bookmark.collectionName === "Comments" && bookmark.comment
+            ? (
+              <QuickTakesListItem
+                key={bookmark._id}
+                quickTake={bookmark.comment}
+              />
+            )
+            : null
+        )}
+        {bookmarks.length === 0 && !loading && (
+          <div className={classes.text}>
+            No saved comments yet
+          </div>
+        )}
+      </SingleColumnSection>
+      </OverflowNavObserverProvider>
+      </UltraFeedObserverProvider>
+    </AnalyticsContext>
```

Comment:
I noticed there is no ability to load more. This was already the case in the previous version, but would you mind fixing it here anyway? I also made a [ticket](https://app.asana.com/1/624079082995/project/628521446211730/task/1211064236859636?focus=true) if you want to come back to it later